ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -78,22 +80,23 @@ public static Optional<KsqlEntity> execute(
     }
 
     return response.error()
-        .map(err -> new ErrorEntity(statement.getStatementText(), err));
+        .map(err -> new ErrorEntity(maskedStatementText, err));
   }
 
   private static Optional<KsqlEntity> handleIfNotExists(
       final ConfiguredStatement<CreateConnector> statement,
       final CreateConnector createConnector,
       final ConnectClient client) {
+    final String maskedStatementText = QueryMask.getMaskedStatement(statement.getStatementText());
```

Comment:
nit: if `statement` isn't used for anything else in this method, can we just pass in `maskedStatementText` instead since we already have it from above?